routes/orders/order.model.js

```diff
@@ -47,12 +44,18 @@ OrderSchema.statics = {
             });
     },
 
-    list({ skip = 0, limit = 50 } = {}) {
+    list({skip = 0, limit = 50} = {}) {
         return this.find()
-            .sort({ createdAt: -1 })
+            .sort({createdAt: -1})
             .skip(+skip)
             .limit(+limit)
             .exec();
+    },
+
+    get(orderStatus) {
```

Comment:
Just realised this method overrides the previous `get(id)`, probably want to rename to either or both to `getBy*` or something.